scripts/add_connector.sh

```diff
@@ -1,33 +1,60 @@
 #! /usr/bin/env bash
-
+function find_prev_connector() {
+    self=scripts/add_connector.sh
+    cp $self $self.tmp
+    connectors=(aci adyen airwallex applepay authorizedotnet bambora bluesnap braintree checkout cybersource dlocal fiserv forte forte forte forte forte forte forte forte forte forte forte forte forte forte globalpay klarna mollie multisafepay nuvei payu rapyd shift4 stripe trustpay utils worldline worldpay "$1")
```

Comment:
1. Is this going to be hard-coded, or do you plan to do something along the lines of: "list dirs in `router/src/connectors`, sort everything except `utils` in order"?.
2. `forte` is repeated multiple times.

Reply:
1. it is going to be hardcoded, whenever some add a new connector,it will update this file also automatically
2. will fix it